{"uuid":"d953105e-3a64-4ffc-964e-0dce3b015ea5","name":"An uncaught error was detected outside of a test","historyId":"36e33adbaf50b4f862768cd0ad9899e3:d41d8cd98f00b204e9800998ecf8427e","status":"broken","statusDetails":{"message":"The following error originated from your test code, not from Cypress.\n\n  > _savedSearchesPage.default is not a constructor\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","trace":"TypeError: The following error originated from your test code, not from Cypress.\n\n  > _savedSearchesPage.default is not a constructor\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at eval (webpack://ggmstestautomation/./cypress/e2e/saved-searches.cy.js:22:24)\n    at eval (https://mergestack-com.site-dev.ggms.com/__cypress/tests?p=cypress/e2e/saved-searches.cy.js:260:3)\n    at eval (https://mergestack-com.site-dev.ggms.com/__cypress/tests?p=cypress/e2e/saved-searches.cy.js:262:12)\n    at eval (<anonymous>)\n    at <unknown> (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:110902:23)\n    at tryCatcher (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:1777:23)\n    at Object.gotValue (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:6446:18)\n    at Object.gotAccum (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:6435:25)\n    at Object.tryCatcher (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:1777:23)\nFrom previous event:\n    at Promise.longStackTracesCaptureStackTrace [as _captureStackTrace] (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:3456:19)\n    at Promise._then (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:1209:17)\n    at Promise.then (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:1102:17)\n    at runScriptsFromUrls (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:110906:136)\n    at Object.runScripts (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:110947:12)\n    at $Cypress.onSpecWindow (https://mergestack-com.site-dev.ggms.com/__cypress/runner/cypress_runner.js:41404:67)\n    at <unknown> (https://mergestack-com.site-dev.ggms.com/__cypress/iframes/cypress%2Fe2e%2Fsaved-searches.cy.js?browserFamily=chromium:275:22)"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"labels":[{"name":"language","value":"javascript"},{"name":"framework","value":"cypress"},{"name":"host","value":"macbooks-MacBook-Air.local"},{"name":"thread","value":"pid-15055-worker-main"},{"name":"package","value":"cypress.e2e.saved-searches.cy.js"}],"links":[],"start":1747639379979,"fullName":"cypress/e2e/saved-searches.cy.js#An uncaught error was detected outside of a test","testCaseId":"36e33adbaf50b4f862768cd0ad9899e3","stop":1747639379994}